OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java

```diff
@@ -1809,6 +1814,37 @@ public String getRef() {
 			return null;
 		}
 
+		public String getOuterRadius() {
+			String outerRadius = "";
+			QuadRect rect = getRect();
+			if (rect.width() > 0.00001 && rect.height() > 0.00001) {
+				WptPt wptPt = findPointToShow();
+				if (wptPt != null) {
+					int radius = (int) Math.max(MapUtils.getDistance(wptPt.lat, wptPt.lon, rect.bottom, rect.left),
```

Comment:
this should be radius of whole gpx (max distance inside gpx) not distance between wpt and corner